prov/efa/src/rdm/rxr_pkt_type_misc.c

```diff
@@ -305,11 +305,11 @@ void rxr_pkt_handle_readrsp_sent(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_en
 	rx_entry->window -= data_len;
 	assert(rx_entry->window >= 0);
 	if (rx_entry->bytes_sent < rx_entry->total_len) {
-		/*
-		 * We currently require EFA RDMA support to enable the FI_HMEM
-		 * devices we support. Emulated read should not be used.
+		/**
+		 * Emulated read should not be used on FI_HMEM
+		 * unless the domain claims RDMA support.
 		 */
-		assert(!efa_mr_is_hmem(rx_entry->desc[0]));
+		assert(efa_domain_support_rdma_read(rxr_ep_domain(ep)) || !efa_mr_is_hmem(rx_entry->desc[0]));
```

Comment:
I am not sure these two things are related.

Reply:
Maybe just remove the comment entirely.

Reply:
Thanks. Address.

Could you also explain your reasoning that `efa_domain_support_rdma_read` is not relevant here?